### PR TITLE
Add New South Wales's Legislation Explorer

### PR DESCRIPTION
**Pull request by maukoquiroga**
Capture d’écran 2020-01-11 à 12 51 05

### CODE REVIEW

File: content/_index.en.md [hunk 1]
```diff
@@ -24,4 +24,8 @@ usecases:
     title: Explore the Rules
     description: Model any kind of government rule, explore them and evaluate who is eligible to what.
     caption: (Exploring the Rules) is a web interface to explore the rules in New Zealand's Openfisca built from government legislation, regulation and some policies.
+
+  legexp_nsw:
+    caption_1: uses OpenFisca and its
```
Comment (Br3nda) [inline content/_index.en.md]: This sentence doesn't complete

Comment (maukoquiroga) [reply content/_index.en.md]: Thanks @Br3nda ! Take a look at the html code and the image posted in the description: it is counter-intuitive, but those variables are used to build a sentence in the final html.

File: layouts/index.html [hunk 2]
```diff
@@ -72,6 +72,15 @@ <h3>{{ $.Param "usecases.rapu_ture.title" }}</h3>
 			</figure>
 		</section>
 
+		<section class="row">
+			<div class="left-col"></div>
+
+			<figure>
+				<img class="screenshot" src="/img/legexp-nsw.png" alt="New South Wales's Legislation Explorer">
+				<figcaption><a href="http://nsw-rules.herokuapp.com/">New South Wales</a> {{ $.Param "usecases.legexp_nsw.caption_1" }} <a href="https://github.com/openfisca/legislation-explorer">Legislation Explorer</a> {{ $.Param "usecases.legexp_nsw.caption_2" }}</figcaption>
```
Comment (maukoquiroga) [inline layouts/index.html]: @Br3nda here.

**Comment by maukoquiroga** [comment]
@tjharrop @UgoHarry @asgharkazi I went ahead and merged, please let us know if you'd like to improve or modify this changes.

**Pull request merged**